src/test/java/com/powsybl/openloadflow/network/ConnectedComponentNetworkFactory.java

```diff
@@ -14,6 +14,18 @@
  * @author Gael Macherel <gael.macherel at artelys.com>
  */
 public class ConnectedComponentNetworkFactory extends AbstractLoadFlowNetworkFactory {
+
+    /**
+     * b1 ----------+
```

Comment:
Could you include the `<PRE>` `</PRE>` tags around the network drawings? so that it's displayed properly in the javadoc